src/auro-dropdown.js

```diff
@@ -94,7 +94,32 @@ class AuroDropdown extends LitElement {
     this.dropdownWidth = this.getBoundingClientRect().width;
   }
 
+  /**
+   * @private
+   * @returns {void} Updates the trigger `aria-label` or `aria-labeledby`.
+   */
+  processAriaLabel() {
+    this.trigger = this.shadowRoot.querySelector(`#trigger`);
+
+    if (this.getAttribute('aria-label')) {
+      // If custom aria-label declared at root apply it to the trigger
+      this.trigger.setAttribute('aria-label', this.getAttribute('aria-label'));
+    } else if (this.getAttribute('aria-labeledby')) {
```

Comment:
This attribute is misspelled - it's [aria-labelledby](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-labelledby)